source/jobs/worker_framework.rst

```diff
@@ -126,168 +128,172 @@ It has to contain the names of the variables on the first line, followed
 by 100 parameter instances in the current example. Items on a line are
 separated by commas.
 
-The job can now be submitted as follows:
-
-::
+The job can now be submitted as follows::
 
-   $ module load worker/1.5.0-intel-2014a
-   $ wsub -batch run.pbs -data data.txt
+   $ module load worker/1.6.10-intel-2018a
+   $ wsub  -batch run.pbs  -data data.txt
 
-Note that the PBS file is the value of the -batch option . The cfd-test
+Note that the PBS file is the value of the -batch option . The ``cfd-test``
 program will now be run for all 100 parameter instances—7
 concurrently—until all computations are done. A computation for such a
-parameter instance is called a work item in Worker parlance.
+parameter instance is called a work item in worker parlance.
+
 
 .. _job arrays:
 
-Job arrays
-~~~~~~~~~~
+Job array-like scenario
+~~~~~~~~~~~~~~~~~~~~~~~
 
-Worker also supports job array-like usage pattern since it offers a
+worker also supports job array-like usage pattern since it offers a
 convenient workflow.
 
-A typical PBS script run.pbs for use with job arrays would look like
-this:
-
-::
+A typical PBS script ``run.pbs`` for use with job arrays would look like
+this::
 
    #!/bin/bash -l
    #PBS -l nodes=1:ppn=1
    #PBS -l walltime=00:15:00
+
    cd $PBS_O_WORKDIR
-   INPUT_FILE=\"input_${PBS_ARRAYID}.dat\"
-   OUTPUT_FILE=\"output_${PBS_ARRAYID}.dat\"
-   word-count -input ${INPUT_FILE}  -output ${OUTPUT_FILE}
+   INPUT_FILE="input_${PBS_ARRAYID}.dat"
+   OUTPUT_FILE="output_${PBS_ARRAYID}.dat"
+   word-count  -input "${INPUT_FILE}"  -output "${OUTPUT_FILE}"
 
 As in the previous section, the word-count program does not exist. Input
 for this fictitious program is stored in files with names such as
-input_1.dat, input_2.dat, ..., input_100.dat that the user produced by
-whatever means, and the corresponding output computed by word-count is
-written to output_1.dat, output_2.dat, ..., output_100.dat. (Here we
-assume that the non-existent word-count program takes options -input and
--output.)
+``input_1.dat``, ``input_2.dat``, ..., ``input_100.dat`` that the user
+produced by whatever means, and the corresponding output computed by
+``word-count`` is written to ``output_1.dat``, ``output_2.dat``, ...,
+``output_100.dat``. (Here we assume that the non-existent ``word-count``
+program takes options ``-input`` and ``-output``.)
 
-The job would be submitted using:
+The job would be submitted using::
 
-::
-
-   $ qsub -t 1-100 run.pbs
+   $ qsub  -t 1-100  run.pbs
 
 The effect was that rather than 1 job, the user would actually submit
 100 jobs to the queue system (since this puts quite a burden on the
-scheduler, this is precisely why the scheduler doesn't support job
-arrays).
+scheduler, this is precisely why the use of job arrays is somewhat
+discouraged.
```

Comment:
maybe replace with this (and maybe add a link to the faq 'worker or atools')?
discouraged for very large number of jobs and/or very short jobs.